src/debugger/Debugger.js

```diff
@@ -71,8 +75,8 @@ export class DebugServer {
   // Checking if the debugger needs to take any action on reaching this ast node
   checkForActions(ast: BabelNode) {
     this.checkForBreakpoint(ast);
-
-    // last step: set the current location as the previously executed line
+    this.checkStepIn(ast);
```

Comment:
It's fine right now, but you will soon change it to checkStepComplete() general method.